### ISSUE TITLE
Should `stamp/core/mergeOne()` handle `undefined` as its first parameter?

### ISSUE THREAD

**Issue by PopGoesTheWza** [stampit-org/stamp#55]
Current implementation of `@stamp/core/mergeOne()` does not fully check if its first parameter `dst` is `undefined`. If the second parameter `src` is an object, `dst` is never check and assumed to be an array/object

- `stamp/packages/collision/index.js` line 33

This issue is triggered by unit tests and `standardiseDescriptor()` (in `packages/it/index.js` lines:41, 47 and 56

This issue relates with #52

**Comment by koresar** [comment]
Yeah, you're right. It should handle the first `undefined` argument.

**Comment by PopGoesTheWza** [comment]
I can PR this (provided I get branching rights)

**Comment by koresar** [comment]
I think I have provided all the member the Write access to all stampit-org repos few minutes ago. Could you please check if I did it right?

**Comment by PopGoesTheWza** [comment]
It looks ok now. Thanks.

**Comment by koresar** [comment]
Outdated
